bin/do_deltas.py

```diff
@@ -82,6 +82,12 @@ def cont_fit(data):
     parser.add_argument('--dla-mask',type=float,default=0.8,required=False,
         help='Lower limit on the DLA transmission. Transmissions below this number are masked')
 
+    parser.add_argument('--absorber-vac',type=str,default=None,required=False,
+        help='Absorber catalog file')
+
+    parser.add_argument('--absorber-mask',type=float,default=2.5,required=False,
+        help='Number of spectral pixels to mask on each side of the absorber central observed wavelength')
```

Comment:
Regarding the discussion of `1e4`, I would change the help to something like `mask width in units of dlambda/lambda*1e-4`

Reply:
I agree that it would be a bit constraining to put it in the catalog, I'm fine with adding a command line option.